Assets/BidMachine/Api/BannerRequest.cs

```diff
@@ -16,7 +17,7 @@ public BannerSize GetSize()
             return client.GetSize();
         }
 
-        public string GetAuctionResult()
+        public AuctionResult GetAuctionResult()
```

Comment:
Don't just change the return type of the object in the public API. Let's deprecate 
```csharp
public string GetAuctionResult()
```
and add new method:
```csharp
public AuctionResult GetAuctionResultObject()
```